<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitializeSyntheticVolumeWidget</class>
 <widget class="QWidget" name="InitializeSyntheticVolumeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>230</height>
   </rect>
  </property>
  <property name="flat" stdset="0">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="widgetLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="1">
    <widget class="DataArraySelectionWidget" name="statsArrayWidget" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="DataArraySelectionWidget" name="phaseTypesWidget" native="true"/>
   </item>
   <item row="4" column="1">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="m_ZPoints">
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="m_YPoints">
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="m_XPoints">
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_15">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_18">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_16">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QDoubleSpinBox" name="m_XResolution">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.150000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QDoubleSpinBox" name="m_YResolution">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.150000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QDoubleSpinBox" name="m_ZResolution">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.150000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="6" rowspan="4">
      <widget class="QScrollArea" name="m_ShapeTypeScrollArea">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="m_ShapeTypeScrollContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>293</width>
          <height>118</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 12px;
}</string>
       </property>
       <property name="text">
        <string>Voxel Dims</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="2">
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 12px;
}</string>
       </property>
       <property name="text">
        <string>Spacing (micron)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_171">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 12px;
}</string>
       </property>
       <property name="text">
        <string>Shape Types</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string>QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Estimated No. Grains</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="m_EstimatedGrains">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5" rowspan="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="4">
      <widget class="Line" name="line_2">
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>HDF5 Stats File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFSDropLineEdit" name="m_InputFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_InputFileBtn">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="DataArraySelectionWidget" name="crystalStructuresWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFSDropLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QFSDropLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>DataArraySelectionWidget</class>
   <extends>QWidget</extends>
   <header>SIMPLViewWidgetsLib/FilterParameterWidgets/DataArraySelectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_InputFile</tabstop>
  <tabstop>m_XPoints</tabstop>
  <tabstop>m_YPoints</tabstop>
  <tabstop>m_ZPoints</tabstop>
  <tabstop>m_XResolution</tabstop>
  <tabstop>m_YResolution</tabstop>
  <tabstop>m_ZResolution</tabstop>
  <tabstop>m_ShapeTypeScrollArea</tabstop>
  <tabstop>m_InputFileBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
